lambda-code/submission/src/lib/fileUpload.ts

```diff
@@ -54,7 +56,8 @@ export function findAttachedFileReferencesInSubmissionResponses(
 
 export async function generateFileAccessKeysAndUploadURLs(
   submissionId: string,
-  fileReferences: FileReference[]
+  fileReferences: FileReference[],
+  fileChecksums?: Record<string, string>
```

Comment:
With the previous check on `fileChecksums` in the main function I think we can safely remove the typescript optional `?`.